app/views/searches/show.html.erb

```diff
@@ -7,7 +7,7 @@
 <% end %>
 <%= link_to t("advanced_search"), advanced_search_path, class: "t-link--gray t-link--has-arrow" %>
 <% if @yanked_filter %>
-  <% @subtitle = t('.subtitle', :query => content_tag(:em, h(params[:query]))) %>
```

Comment:
Any reason to not put `html_safe` to content tag? Does this change the markup to wrap whole part with "em" tag?

Reply:
Yeah, that's right. My bad. For previous behaviour it does indeed need a `html_safe` and pass that into the translation.
It looks the same since this is all wrapped in `<i>` but the semantic meaning is different. I blame rubocop for yelling at me for using `html_safe` (:

Reply:
Of course justing moving html_safe to someplace else makes `better_html` mad. I've moved the markup into the translation itself instead, I'm not sure if you could do this someway else instead.

If you have some other approach, feel free to make your own PR, don't want to waste your time for this simple change.

Reply:
No worries, you're not wasting my time, you're actually helping to fix this in the best possible way. Anything keeping the same markup and linters happy at the same time works for me.